src/components/Navbar.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+export default function Navbar() {
+  // [{ name: Home, link: '/' }];
```

Comment:
- [ ] [Optional] Kindly remove the commented code.

Reply:
### Thank you @Deepakdanger I'll make sure to correct the optional suggestion before maraging the pull request.